app/lib/shared/service_utils.dart

```diff
@@ -272,6 +278,17 @@ Future initDartdoc(Logger logger) async {
   }
 }
 
+Future initPkgDartdoc(Logger logger) async {
+  logger.info('Initializing pkg/pub_dartdoc');
+  final dir = Platform.script.resolve('../../pkg/pub_dartdoc').toFilePath();
+  final pr = await Process.run('pub', ['get'], workingDirectory: dir);
```

Comment:
Perhaps have a time out if the command doesn't halt.

Reply:
Good point, adding it.